src/pydap/wsgi/app.py

```diff
@@ -21,6 +21,7 @@
 from datetime import datetime
 import shutil
 
+from gunicorn.app.wsgiapp import WSGIApplication
```

Comment:
Looks like the tests failed for this PR because `gunicorn` is not available for the module to import. I would guess including it in the `install_requires` for `setup.py` would probably mitigate the issue.